lib/evmone/instructions.hpp

```diff
@@ -898,6 +932,22 @@ evmc_status_code create_impl(StackTop stack, ExecutionState& state) noexcept;
 inline constexpr auto create = create_impl<OP_CREATE>;
 inline constexpr auto create2 = create_impl<OP_CREATE2>;
 
+inline code_iterator callf(StackTop /*stack*/, ExecutionState& state, code_iterator pos) noexcept
+{
+    const auto index = read16beu(&pos[1]);
```

Comment:
Cannot this use an iterator? The input would be then `read16beu(pos + 1)`.

Reply:
I'd also would vote for something like `read_int16_be` and `read_uint16_be` for naming. (Maybe skip the second underscore.)

Reply:
Not sure what you mean by "iterator". In general, iterator is not a single type so this would need to be a template. If you mean the usage side, we the `pos + 1` can be used already. Although some C++ guidelines suggest to avoid pointer arithmetic in favor of subscripts.